Cython/Compiler/Dataclass.py

```diff
@@ -635,16 +641,16 @@ def _set_up_dataclass_fields(node, fields, dataclass_module):
                 global_scope.mangle(Naming.dataclass_field_default_cname, node.class_name),
                 name)
             # create an entry in the global scope for this variable to live
-            nn = ExprNodes.NameNode(f_def.pos, name=EncodedString(module_field_name))
-            nn.entry = global_scope.declare_var(nn.name, type=f_def.type or PyrexTypes.unspecified_type,
-                                                pos=f_def.pos, cname=nn.name, is_cdef=1)
+            name_node = ExprNodes.NameNode(field_default.pos, name=EncodedString(module_field_name))
```

Comment:
I think `field_node` or even `field_name_node` is a better name here. Yes, it's some `NameNode`, but I can quickly see that from the code. The question is what the meaning of that `NameNode` is, what it refers to.